src/tsdb.c

```diff
@@ -872,6 +872,20 @@ size_t SeriesDelRange(Series *series, timestamp_t start_ts, timestamp_t end_ts)
     RedisModule_DictIteratorStop(iter);
 
     CompactionDelRange(series, start_ts, end_ts);
+
+    // Check if last timestamp deleted
+    if (end_ts >= series->lastTimestamp) {
+        iter = RedisModule_DictIteratorStartC(series->chunks, "$", NULL, 0);
+        currentKey = RedisModule_DictNextC(iter, &keyLen, (void *)&currentChunk);
+        if (!currentKey) {
+            // No samples in the series
+            series->lastTimestamp = 0;
+            series->lastValue = 0;
```

Comment:
@OfirMos can we extend the test on ts_delete to include also the case in which the series is now empty after the delete?